dijet/config/config_run2.py

```diff
@@ -124,7 +124,8 @@ def add_config(
     # category groups for conveniently looping over certain categories
     # (used during plotting)
     cfg.x.category_groups = {
-        "default": ["incl"]
+        "default": ["incl"],
+        "binning": ["eta_0p5_1p0__pt_100_500__alpha_smaller_0p1", "eta_0p5_1p0__pt_100_500__alpha_smaller_0p3"],
```

Comment:
This should probably be changed to reflect the renaming of inclusive alpha categories;
```suggestion
        "binning": ["eta_0p5_1p0__pt_100_500__alpha_lt_0p1", "eta_0p5_1p0__pt_100_500__alpha_lt_0p3"],
```